doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -679,6 +679,9 @@ Modem libraries
       The application uses the Kconfig option :kconfig:option:`CONFIG_LTE_RAI_REQ` and ``SO_RAI_*`` socket options instead.
     * The CE level enum names for :c:enum:`lte_lc_ce_level` to not include the number of repetitions.
     * The default network mode from :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M` to :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_GPS`.
+    * The ``CONFIG_LTE_MODE_PREFERENCE`` Kconfig option has been renamed to :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE_VALUE`.
+    * The ``CONFIG_LTE_NETWORK_DEFAULT`` Kconfig option has been renamed to :kconfig:option:`CONFIG_LTE_NETWORK_MODE_DEFAULT`.
+    * The LTE mode preference Kconfig choice has been added a name, :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE`.
```

Comment:
```suggestion
    * The LTE mode preference Kconfig choice has been named as ``CONFIG_LTE_MODE_PREFERENCE``.
```
This is a bit confusing. You just renamed this Kconfig option but then you use it as a choice name. Not sure if these choice names can be tagged with :kconfig:option:.

Reply:
Yes, renaming was done to be able to name the choice logically. `CONFIG_LTE_MODE_PREFERENCE_VALUE` is only used by LTE LC internally and modem_shell, it's probably not used widely anyway. And I'm not sure either if the choice name can be referenced, we'll see.

Reply:
At least the doc build was successful.